src/io/profile.cc

```diff
@@ -35,25 +35,25 @@ static char const* trueWords[] = {
    "true",
    /** TRANSLATORS: A variant of the commandline parameter "true" value */
    /** TRANSLATORS: Needs to be consistent with the translations in widelands-console */
-   _("true"), "yes",
+   gettext_noop("true"), "yes",
```

Comment:
I think the plain English entries can be removed. With `gettext_noop()` they result in duplicates.